gtsfm/utils/geometry_comparisons.py

```diff
@@ -194,3 +194,49 @@ def compare_global_poses(
             for (wTi, wTi_) in zip(wTi_list, wTi_list_)
         ]
     )
+
+
+def compute_relative_rotation_angle(
+    R_1: Optional[Rot3], R_2: Optional[Rot3]
+) -> float:
+    """Compute the angle between two rotations.
+
+    Note: the angle is the norm of the angle-axis representation.
+
+    Args:
+        R_1: the first rotation.
+        R_2: the second rotation.
+
+    Returns:
+        the angle between two rotations.
+    """
+
+    if R_1 is None or R_2 is None:
+        return np.pi
```

Comment:
should this be a module-level private global default?

`MAX_ROTATION_ERROR = np.pi`?